src/PlaneWaveBasis.jl

```diff
@@ -236,7 +239,8 @@ end
 
 # This is the "internal" constructor; the higher-level one below should be preferred
 @timing function PlaneWaveBasis(model::Model{T}, Ecut::Number,
-                                kcoords::AbstractVector, ksymops, symmetries=nothing;
+                                kcoords::AbstractVector, ksymops,
+                                symmetries = symmetries_preserving_kcoords_ksymops(model.symmetries, kcoords, ksymops);
```

Comment:
As you know long lines make me uneasy. How about setting this to nothing and inferring the symmetries inside as we did before?

Reply:
That's super ugly, we're not going to not use default argument values just because it makes for long lines. I've broken the line, but of course the issue is that symmetries_preserving_kcoords_ksymops is a ridiculous name. How about symmetries_preserving_BZ(symmetries, kcoords) and symmetries_preserving_BZ(symmetries, kcoords, ksymops)?

Reply:
I guess we wait with the renaming until you are done with the other refactoring. But I'm ok with `symmetries_preserving_bz`

Reply:
I went with preserving_kgrid (because the symmetries always preserve the bz, it's just the discrete one, ie the grid, that is not preserved)

Reply:
yeah, ok. That's not a bad name indeed.